recipes/geocube/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "0.0.2" %}
+
+package:
+  name: geocube
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/g/geocube/geocube-{{ version }}.tar.gz
+  sha256: ef74e677510ab9ef17bb855d65c187fbd50098a5697effdd4749b9cd5c516d79
+
+build:
+  number: 0
+  noarch: python
+  # entry_points:
+  #   - geocube=geocube.cli.geocube:geocube
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - appdirs
+    - click >=6.0
+    - datacube >=1.6.2,<1.7
+    - geopandas
+    - python
+    - rasterio
+    - scipy
```

Comment:
scipy can be replaced with rioxarray for the current version (0.0.3).

Reply:
Done.